playbook/workshop_project_model.md

```diff
@@ -0,0 +1,74 @@
+# Club Structure
+
+People seem to be more motivated when they get to choose what they do.
+Therefore, a Hack Club would ideally be formatted like a maker space. Hack Club
+would be a space for people to come in and hack on projects together. However
+this does not work for complete beginners because they don't know where to
+start.
+
+One solution is only to teach them the basics through lessons or workshops for
+the first several months. That's about as exciting as math teachers saying you
+have to take years of math before you get to do anything interesting with it. We
+want beginners to be able to build their own project as soon as possible.
+Therefore we want them to learn just enough and then as soon as possible, have
+them actually build their own project. This builds confidence and motivation in
+beginners.
+
+At [Hack Camp](https://camp.hackedu.us/), we repeated a 2-day cycle where:
+
+1. learners spend the first 2 hours of their first day following a workshop
+2. for the remainder of the first day and all of the second day, they would
+   come up with and build their own projects [the details are documented here](https://github.com/hackedu/hack-camp/blob/master/cohort_4/playbook/SCHEDULE.md).
+
+We think a similar division can work for clubs:
+
+**Workshop Time**
+
+Have one club meeting where a beginning hacker learns to ship a project by
+following a tutorial.
+
+**Project Time**
+
+Let them work on their own project at the 1-3 club meetings before starting
+their next tutorial.
+
+## Workshop Time F.A.Q
+
+### So should my workshops be lecture based?
+
+Most likely not. People learn differently and at different rates. If you just
+talk at them, that might work for one person in the room but everyone else will
+need you to go faster and slower.
+
+There should be no point ever in a club where this is one person speaking for
+more than 10 minutes in a row or else people get bored.
+
+### So what should I do?
+
+Our workshops can be found
+[here](https://github.com/hackedu/hackedu/tree/master/playbook/workshops)
+
+They are designed so that by the end of every workshop, you can ship something
+tangible that you would want to show your friends.
+
+There should be a new workshop released every week.
+
+### Should everyone be working on the same workshop?
+
+People will want to work on different things and work at different rates.
+Let them.
+
+## Project Time Pro Tips
+
+### "I don't know what to do?"
+
+Many people, when asked what project they want to work on, say they don't know.
+
+Doing a brainstorming session before this helps a lot.
+[Here is an example](https://github.com/hackedu/hack-camp/blob/master/cohort_4/playbook/ACTIVITIES.md#running-a-brainstorm-session)
+of how you might run a brainstorming session.
+
+### How do I run freeform project time?
+
+Here is a much more
+[comprehensive guide](https://github.com/hackedu/hack-camp/blob/master/cohort_4/playbook/ACTIVITIES.md#freeform-projects)
```

Comment:
Maybe move this into the hackedu/hackedu repository and make it a bit more specific to the code club environment